openssl/math.go

```diff
@@ -176,3 +176,51 @@ func DupBN(from BigNum) (BigNum, error) {
     }
     return bn, nil
 }
+
+func CmpECP(group ECGroup, a, b ECPoint, ctx BNCtx) (bool, error) {
```

Comment:
All these functions below have the EC group as an argument, but can you just take it from the points' curve?

Reply:
`goUmbral/openssl` cannot import `goUmbral/math` because that would create a circular dependency in Go. So while the function cannot take Points as an argument, they could take the Curve as an argument, since Curve is also in the openssl package.

This was merely a design decision. I wanted a very thin wrapper that handle OpenSSL errors. If a user might want to just import `goUmbral/openssl` for the sake of using those wrappers in a different manner, they can do that now.

Tell me what you think because this remains mostly just a difference in design.